NewPlatform.Flexberry.ServiceBus.RabbitMQ/Components/MessageConverter/RmqMessageConverter.cs

```diff
@@ -14,6 +15,11 @@ internal class MessageConverter : IMessageConverter
 
         private string _senderIdPropepertyName => "senderId";
 
+        /// <summary>
+        /// Свойство сообщения RabbitMQ, в котором хранится Timestamp сообщения
+        /// </summary>
+        protected string TimestampPropertyName => "timestamp_in_ms";

```

Comment:
What is the difference of this property, for example, from `_senderIdPropepertyName`?

Reply:
In derived classes it can be overrided if timestamp is stored in other header.